frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.js

```diff
@@ -101,7 +101,7 @@ export default function NextStepsRepeater({
 
   const dateLabel = (index) => (stepType === 'recipient'
     ? `When does the ${recipientLabel} anticipate completing step ${index + 1}?`
-    : `When do you anticipate completing step ${index + 1}?`);
+    : 'When do you anticipate completing this step?');
```

Comment:
we can't have multiple inputs with the same accessible name, this needs to be the way it was

Reply:
reverted here: f1187bd9